config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -1317,7 +1343,7 @@ presubmits:
         - --test=node
         - --
         - --repo-root=.
-        - --gcp-project=k8s-jkns-pr-node-e2e
+        - --gcp-project=
```

Comment:
```suggestion
```

rm empty flag

Reply:
Sure I will update this PR in a bit with all the suggested changes.